lib/rules/no-multiple-empty-lines.js

```diff
@@ -98,7 +98,7 @@ module.exports = {
 
                 linesRangeStart.push(0);
                 lines.forEach(function(str, i) {
-                    var length = i < fullLines.length ? fullLines[i].length : 0,
+                    var length = Array.isArray(fullLines) && i < fullLines.length ? fullLines[i].length : 0,
```

Comment:
Could an alternative fix be to add `|| []` on line 81 above, so that it's always an array, and you don't have to check if it is one N times?